test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -101,7 +101,11 @@ type workload struct {
 }
 
 type workloadParams map[string]interface{}
-func (wp workloadParams) getInt(key string) (int, bool) { n, ok := wp[key].(float64); return int(n), ok }
+
+func (wp workloadParams) getInt(key string) (int, bool) {
+	n, ok := wp[key].(float64)
```

Comment:
unmarshal always does so with float64?

Reply:
Yes, unfortunately. In fact, with `sigs.k8s.io/yaml`, the YAML is converted to JSON, and the JSON is unmarshalled, so I imagine there's some loss of information in the intermediate steps.

Reply:
do we need floats? Can we use `map[string]int`?

Reply:
we may need bool/string template vars, which is why the map value is an interface{}

Reply:
Switched to int.